.packit.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Packit's default fix-spec-file often doesn't fetch version string correctly
+# and may also not account for new PRs installing additional files. This script
+# handles any custom processing of the dist-git spec file and gets used by the
+# fix-spec-file action in .packit.yaml
+
+# Get Version from HEAD
+export HEAD_VERSION=$(grep 'var Version = semver.MustParse' version/version.go | cut -d\" -f2 | sed -e 's/-/~/')
+
+# Generate source tarball
+git archive --prefix=podman-$HEAD_VERSION/ -o podman-$HEAD_VERSION.tar.gz HEAD
+
+# RPM Spec modifications
+
+# Fix Version
+sed -i "s/Version: .*/Version: $HEAD_VERSION/" podman.spec
```

Comment:
Discomfort: without an anchor (`^`), this could catch `Version: ` anywhere on a line. Unlikely, but a tighter safer expression would be
```bash
set -i "s/^Version:.*/....
```